packages/pools/src/constants/boostedPools/arb.ts

```diff
@@ -0,0 +1,10 @@
+import { BoosterConfig, BoosterType } from '../../utils/boosted/types'
+
+export const arbBoostedPools: BoosterConfig[] = [
+  {
+    boosterType: BoosterType.ALP,
+    contractAddress: '0x0639c5715EC308E16f089c96C0C109302d76FA81',
+    tooltipsText:
```

Comment:
One more suggestion, but let's try refactor next time we have another booster live. The tooltip should also match with different booster type. And it's not necessary to be part of the config. We can put it into the component.